src/main/resources/application-dev.yml

```diff
@@ -10,7 +10,7 @@ logging:
     root: INFO
     com.odysseusinc: DEBUG
 portal:
-   urlWhiteList: https://localhost:${server.port}
+   urlWhiteList: https://localhost:${server.port},https://sandbox.arachnenetwork.com:8433
```

Comment:
Pls, remove any sensitive URLs or other info from public repository
Or if it's required, just replace with something neutral

Reply:
This is non-existent host that one can define in /etc/hosts for testing on local machine.
I agree it ideally shouldn't be here, but I don't see this as a sensitive url.

Reply:
let's just make it `sandbox.domain` or similar

Reply:
Then it is pointless. The url has to be registered with oauth provider to make sense of hardcoding it here. 
Generally it has to have at least top-level domain defined to pass.
It is highly unlikely that something like sandbox.local or sandbox.domain will get through the approval process.